src/main/java/edu/cornell/kfs/pdp/CUPdpParameterConstants.java

```diff
@@ -4,7 +4,18 @@
 
     public static final String WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES = "WARNING_INACTIVE_VENDOR_TO_EMAIL_ADDRESSES";
 
+    public static final String ACH_PERSONAL_CHECKING_TRANSACTION_CODE = "ACH_PERSONAL_CHECKING_TRANSACTION_CODE";
+    public static final String ACH_PERSONAL_SAVINGS_TRANSACTION_CODE = "ACH_PERSONAL_SAVINGS_TRANSACTION_CODE";
+    public static final String ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE = "ACH_DIRECT_DEPOSIT_TRANSACTION_TYPE";
+    public static final String ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS = "ACH_SECURE_FIELDS_ALLOWED_IN_ACCOUNT_EMAILS";

```

Comment:
The users wanted to be able to include the bank name in notification emails, where I added the ability to specify property placeholders in an email message parameter. I also added checks of DD AttributeSecurity to disallow secure fields in the final message; unfortunately, our customizations are masking bank routing number and bank name. Some sort of override was necessary for this strategy to allow for bank name inclusion.

The easiest alternative would be to add a new config property to replace the secure fields parameter, to prevent it from being altered at runtime. Another alternative would be to define a whitelist of specific placeholders that map to BO properties, but that would be a larger effort. I will at least look into implementing the former if you believe that would be enough to alleviate the security concerns.